openstack/resource_openstack_networking_router_v2.go

```diff
@@ -183,6 +199,7 @@ func resourceNetworkingRouterV2Create(ctx context.Context, d *schema.ResourceDat
 			TenantID:              d.Get("tenant_id").(string),
 			AvailabilityZoneHints: resourceNetworkingAvailabilityZoneHintsV2(d),
 		},
+		d.Get("flavor_id").(string),
```

Comment:
this would be better, especially if you're grepping for a place, where the `RouterID` member is set.
```suggestion
		RouterID:   d.Get("flavor_id").(string),
```